distribution/test/itests/test-itests-dependencies-app/src/main/filtered-resources/features.xml

```diff
@@ -12,7 +12,7 @@
  *
  **/
 -->
-<features name="ddf-itest-dependencies">
+<features name="ddf-itest-dependencies-0.0.0">
```

Comment:
The original code was defaulting to 0.0.0 if the version was absent. But it makes sense to use  ${project.version}